### ISSUE TITLE
Only getting jet events despite them being off.

### ISSUE THREAD

**Issue by 4nalMac** [TEHE-Studios/ExpandedHelicopterEvents#353]
I am doing a six month later run and have events scheduled forever but only have survivors and raiders activated but only seem to be getting jets.
 
I'm not getting any kinds of errors but if logs will help I can post them, 
![image](https://github.com/user-attachments/assets/6d437b11-b20d-44f9-a126-ac0954979579)

**Comment by Chuckleberry-Finn** [comment]
The logs may help. Do you have a sandbox.bin in your MP folder? (Assuming this is a MP server?)

**Comment by 4nalMac** [comment]
> The logs may help. Do you have a sandbox.bin in your MP folder? (Assuming this is a MP server?)

This was on a SP save and unfortunately I foolishly deleted my console and logs recently without thinking. I will post them the moment another event happens so that you can look.

**Comment by Chuckleberry-Finn** [comment]
Unless you use a mod, you can only change sandbox settings at the creation of a SP game. Do you have a mod that lets you set sandbox options?

**Comment by 4nalMac** [comment]
Yes, I was using stars change sandbox settings so I can tweak things mid game.

**Comment by 4nalMac** [comment]
I don't think I've touched the settings mid game from what I remember. That should be how it was set since I started. I only opened that menu mid game for the screenshot.
